src/test/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActorTest.scala

```diff
@@ -74,13 +66,15 @@ class TaskLauncherActorTest extends AkkaUnitTest {
       offerMatchStatisticsActor: TestProbe = TestProbe(),
       localRegion: () => Option[Region] = () => None) {
 
+    Mockito.when(instanceTracker.instancesBySpecSync).thenReturn(InstanceTracker.InstancesBySpec.forInstances(f.scheduledInstance))
+
     def createLauncherRef(appToLaunch: AppDefinition = f.app): ActorRef = {
       val props = TaskLauncherActor.props(
         launchQueueConfig,
         offerMatcherManager, clock, instanceOpFactory,
         maybeOfferReviver = Some(offerReviver),
         instanceTracker, rateLimiterActor.ref, offerMatchStatisticsActor.ref, localRegion) _
-      system.actorOf(props(appToLaunch))
+      TestActorRef[TaskLauncherActor](props(appToLaunch))
```

Comment:
This does not work with `stash` in the actor system. We have to use a real actor.